app/policies/feed/apple_subscription_policy.rb

```diff
@@ -0,0 +1,5 @@
+class Feed::AppleSubscriptionPolicy < ApplicationPolicy
+  def show?
```

Comment:
maybe could leave this out until UI PR? i think this got added in a commit before realizing which branch i was in.

Reply:
Yep - I think you can delete this.  You shouldn't need it at all, since you set `self.model_name` to Feed in the model.  That causes the child-model to use the policy of the parent.